src/markdown-pages/explore-docs/nr1-cli.mdx

```diff
@@ -56,7 +56,7 @@ This table gives descriptions and context for some of the most important CLI com
 | Profiles | `nr1 profiles` | Controls the profile you'll be running CLI commands as. You can have more than one profile, which is helpful for executing commands on multiple New Relic accounts. For more on this, see [the workshop](https://github.com/newrelic/nr1-workshop/blob/master/lab-cli/INSTRUCTIONS.md). |
 | Create | `nr1 create` | Create a new component template (Nerdpack, Nerdlet, launcher, or catalog). The CLI will walk you through this process. For an explanation of the file structure, see [Nerdpack file structure](/explore-docs/nerdpack-file-structure). |
 | Clone | `nr1 nerdpack:clone` | Clone an [open source Nerdpack from our GitHub repository](https://opensource.newrelic.com/explore-projects). |
-| Serve locally | `nr1 nerdpack:serve` | Serves your Nerdpack locally, which makes it visible at [one.newrelic.com/?nerdpacks=local](https://one.newrelic.com/?nerdpacks=local). For details, see [Local development](/client-side-sdk/index.html#cli/LocalDevelopment). |
+| Serve locally | `nr1 nerdpack:serve` | Serves your Nerdpack locally, which makes it visible at [one.newrelic.com/?nerdpacks=local](https://one.newrelic.com/?nerdpacks=local). For details on working locally, see our documentation on setting up your [development environment](/build-apps/set-up-dev-env). |
```

Comment:
no - should say "For details on working locally, see our [publish and deploy information]( /build-apps/publish-deploy).